test/parallel/test-child-process-spawn-shell.js

```diff
@@ -2,63 +2,92 @@
 const common = require('../common');
 const assert = require('assert');
 const cp = require('child_process');
+{
```

Comment:
After you squashed the commits, it's no longer clear which changes to this file are part of the PR and which changes are refactoring.

Reply:
The added test from this PR is in line 56
The rest are refactoring for wrapping the other test-cases with curly braces.